notebooks/concepts/imviz_compass_mpl.ipynb

```diff
@@ -0,0 +1,251 @@
+   "outputs": [],
+   "source": [
+    "a = np.arange(100).reshape((10, 10))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c334e59e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "20605e33",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs_utils.draw_compass_mpl(a, w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8999c3b9-2bf3-4b6a-89e9-5f6ec843a73e",
+   "metadata": {},
+   "source": [
+    "## Case 2: Real WCS with no rotation nor distortion\n",
+    "\n",
+    "We reuse the array from above but WCS from https://learn.astropy.org/tutorials/celestial_coords1.html example."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3253dd7f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "w = WCS({'CTYPE1': 'RA---TAN',\n",
+    "         'CUNIT1': 'deg',\n",
+    "         'CDELT1': -0.0002777777778,\n",
+    "         'CRPIX1': 1,\n",
+    "         'CRVAL1': 337.5202808,\n",
+    "         'NAXIS1': 10,\n",
+    "         'CTYPE2': 'DEC--TAN',\n",
+    "         'CUNIT2': 'deg',\n",
+    "         'CDELT2': 0.0002777777778,\n",
+    "         'CRPIX2': 1,\n",
+    "         'CRVAL2': -20.833333059999998,\n",
+    "         'NAXIS2': 10})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d2e73414",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs_utils.draw_compass_mpl(a, w)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "321b9356-abc6-4a8f-b76c-7f9bf284a248",
+   "metadata": {},
+   "source": [
+    "## Case 3: HST/ACS FLT with distortion\n",
+    "\n",
+    "We replaced the actual ACS image (`jb5g05ubq_flt.fits`) with random data to avoid data download, but the WCS is real."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42a9e142",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "a = np.random.random((2048, 4096))\n",
+    "w = WCS({'WCSAXES': 2,\n",
+    "         'CRPIX1': 2100.0,\n",
+    "         'CRPIX2': 1024.0,\n",
+    "         'PC1_1': -1.14852e-05,\n",
+    "         'PC1_2': 7.01477e-06,\n",
+    "         'PC2_1': 7.75765e-06,\n",
+    "         'PC2_2': 1.20927e-05,\n",
+    "         'CDELT1': 1.0,\n",
+    "         'CDELT2': 1.0,\n",
+    "         'CUNIT1': 'deg',\n",
+    "         'CUNIT2': 'deg',\n",
+    "         'CTYPE1': 'RA---TAN',\n",
+    "         'CTYPE2': 'DEC--TAN',\n",
+    "         'CRVAL1': 3.581704851882,\n",
+    "         'CRVAL2': -30.39197867265,\n",
+    "         'LONPOLE': 180.0,\n",
+    "         'LATPOLE': -30.39197867265,\n",
+    "         'MJDREF': 0.0,\n",
+    "         'RADESYS': 'ICRS'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "7bd46c49",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "wcs_utils.draw_compass_mpl(a, w)"
```

Comment:
| This PR | Ginga |
| --- | --- |
| ![hst_acs_flt_wcs_imviz](https://user-images.githubusercontent.com/2090236/144470698-93e2a27f-9f34-4279-837a-9d4e56f5f7fb.jpg) | ![hst_acs_flt_wcs_ginga](https://user-images.githubusercontent.com/2090236/144470727-7a85f832-b325-4b0e-a8b5-1962dad222a0.jpg) |